### ISSUE TITLE
Issue in building the paper

### ISSUE THREAD

**Issue by ashutoshsaboo** [sympy/sympy-paper#173]
Hello, 

When I execute `make` in the sympy-paper's source directory, the following error pops up-:

```
ashutoshsaboo@ASHUTOSH-PC:~/sympy-paper$ make
cd authors; ./list_latex.py
xelatex --halt-on-error pprint.tex
make: xelatex: Command not found
Makefile:19: recipe for target 'pprint' failed
make: *** [pprint] Error 127

```

Actually I was a bit busy since about a week, and hence thought must rebuild the PDF to track the changes in the paper, and review it again. 

How do I resolve this error?

Thanks! :smile:

**Comment by scolobb** [comment]
You seem to be missing `xelatex`.

Do you have a full TeX Live distribution?  It is usually packaged under names like texlive-full in popular distros.

**Comment by ashutoshsaboo** [comment]
Yeah I did install the full package only @scolobb . But i don't know, some problem might have happened. 

Now how do I install the `xelatex` package? Could you help on that @scolobb ?

Thanks :smile:

**Comment by scolobb** [comment]
Hmm... Tell us what distro you are on and what TeX Live package you have installed.  You might want to try uninstalling and installing TeX Live (sorta uselessly obvious advice :-) ).

What happens if you just type `xelatex` in a terminal?

**Comment by hargup** [comment]
@ashutoshsaboo if you are on ubuntu try `sudo apt-get install texlive-full`.
